### DISCUSSION TITLE
Wayland native backend

### DISCUSSION THREAD

**Discussion by phlash** [openscenegraph/OpenSceneGraph#1260] [General]
Hi folks,

I have just submitted a PR (#1170) that adds my native Wayland graphics backend for OSG. Since I expect there to be some opinions on how this works and the Wayland features we might want supported.. please use this topic :smile_cat: 

Cheers,
Phil.

**Comment by robertosfield** [comment]
I have so much pressing work to complete on the VulkanSceneGraph side that I won't be able to pop back to do OSG support work for a while.  There is slim chance that I might have an opportunity later in the year, but even then I'm unlikely to be able to clear the back log of PRs.  This means I won't be able to review and comment on your PR for the near term.  

As a general note, the OSG is maintenance mode - so keeping existing features working is the priority, new feature development will need to tackled only when all the maintenance work is complete.

On the VulkanSceneGraph side I would like to see Wayland support added and have looked a few implementations such as glfw's Wayland backend so have seen a few of the things that need to be tackled.  It would be great to see it added before end of year, but then there are lots of features I'd like to see completed before end of year so it's not something that will bubble to the front to task queue right away.

**Reply by phlash** [reply]
No problem Robert.. I was interested to get some feedback, hence this topic.
